Fantasy-Quest/Assets/Scripts/Dialogue/DialogueSpeaker.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using Sirenix.OdinInspector;
+using Subtitles;
+using UnityEngine;
+
+namespace Dialogue
+{
+    [RequireComponent(typeof(Collider2D))]
+    [RequireComponent(typeof(AudioSource))]
+    [AddComponentMenu("Scripts/Dialogue/Dialogue.DialogueSpeaker")]
+    public class DialogueSpeaker : MonoBehaviour, ISpeakable
+    {
+        [Header("Speech")]
+        [SerializeField]
+        private List<Replica> firstTrySpeech;
+
+        [SerializeField]
+        private List<Replica> alternativeSpeech;
```

Comment:
Maybe move this lists to SO config woud be good, and here store link to SO.

Reply:
I don't think it's necessary